manifest.toml

```diff
@@ -54,8 +54,8 @@ ram.runtime = "50M"
         autoupdate.strategy = "latest_github_tag"
      
         [resources.sources.addons]
-        url = "https://github.com/friendica/friendica-addons/archive/refs/tags/2023.05.tar.gz"
-        sha256 = "df29aed28c0208e162c76f91949d0c3a0f77fe09853fe1a6d854b9956075d8a0"
+        url = "https://github.com/friendica/friendica/archive/refs/tags/2023.05.tar.gz"
+        sha256 = "727a8fdab6a2f6424d3dbc895496447e750eb0f8e1e11b70f1e229a7e3c9a31d"
```

Comment:
This looks wrong, there are no addons in the `friendica/friendica` repository on GitHub.